apps/sample-configs/osvr_server_config.moverio.json

```diff
@@ -0,0 +1,98 @@
+{
+  "description": "This configuration supports video (so-called 'positional') and IMU fusion tracking, in addition to orientation-only tracking, with the OSVR HDK 2, for units that were manufactured as HDK2 (NOT UPGRADED FROM HDK 1.x!). It is configured for RenderManager applications in direct mode on HDK2 optics.",
+  "display": {
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "Epson",
+      "model": "Moverio BT-300 series",
+      "num_displays": 1,
+      "Version": "1.0",
```

Comment:
This is actually hardware version, IIRC - I think you can actually leave it out if it's not meaningful.